### PR TITLE
Account permissions to a publication 

### LINKED ISSUE #8: Make it possible to give an account permissions to a publication
An account that has the `publication/permissions` permission should be able to give another account permissions to the publication via the `publication/permissions` action (https://github.com/gnosis/tabula#set-publication-permissions).

Display the same form as when creating an article, just pre-populate it with the article to be updated.

- [x] Subgraph
- [x] App
- [x] Design

### PR DESCRIPTION
**Pull request by juliopavila**
closed #8 

Completed tasks

- [x] Create Publication Permission UI for Add/Update/Delete Permission.
- [x] Integrate permissions actions with the contract
- [x] Add a poll interval after success transaction to improve the UX by redirecting the user to the proper screen after the transaction is already indexed.
- [x] If the user press "Remove User." we hit the contract call and set all fields (checkboxes) to false and then only show the users that have some permissions on true.

Tests:
https://rinkeby.etherscan.io/tx/0x6d251b155a7f497fe72fde33203f1ccc9de8edf519445d05bdfe3c2a5c1a9128
https://rinkeby.etherscan.io/tx/0x84839ab35825c78c990139b79302f97405862eb72716b6596d2ed88d2339536b
https://rinkeby.etherscan.io/tx/0xf31edb1965d6de4ffae0a3fd578cf5597eac687a3b907a8ef440200da64b8ce6

Screens:

image
image
image

### CODE REVIEW

File: packages/app/src/components/commons/PermissionItem.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const { address, id } = permission
+
+  return (
+    <PermissionItemContainer container alignItems={"center"} onClick={() => onClick(id ? id : "")}>
+      <Grid item xs={10}>
+        <WalletBadge address={address} />
+      </Grid>
+      <PermissionItemIconGrid item xs={2}>
+        <PermissionItemEditContainer>
+          <EditIcon style={{ color: palette.grays[1000] }} />
```
Comment (asgeir-eth) [inline packages/app/src/components/commons/PermissionItem.tsx:52]: We should probably not show the edit button here to users that do not have the "Can edit the publication permissions" / `publication/permissions`

File: packages/app/src/components/views/publication/components/PermissionSection.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            fontFamily={typography.fontFamilies.sans}
+            sx={{ margin: 0 }}
+          >
+            Permissions
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Button variant="contained" size="medium" onClick={() => navigate("/publication/permission/new")}>
+            <AddIcon style={{ marginRight: 13 }} />
+            New Permission
```
Comment (asgeir-eth) [inline packages/app/src/components/views/publication/components/PermissionSection.tsx:31]: We should probably not show this either when the user does not have the "Can edit the publication permissions" / publication/permissions

**Review by asgeir-eth — CHANGES_REQUESTED** [review]
Clean code! 

Just need to make sure a user does not see the permission administration functionality when not appropriate

File: packages/app/src/components/commons/PermissionItem.tsx (hunk 1 above)
Comment (juliopavila) [reply packages/app/src/components/commons/PermissionItem.tsx:52]: Done 🚀

image

File: packages/app/src/components/views/publication/components/PermissionSection.tsx (hunk 2 above)
Comment (juliopavila) [reply packages/app/src/components/views/publication/components/PermissionSection.tsx:31]: With the other comment, this validation is solved. If the user doesn't have  "publication/permissions," they can't enter on this screen.

Comment (asgeir-eth) [reply packages/app/src/components/views/publication/components/PermissionSection.tsx:35]: @juliopavila: It was this button I menth:  😅  
Screenshot 2022-04-13 at 14 49 41

Comment (asgeir-eth) [reply packages/app/src/components/views/publication/components/PermissionSection.tsx:35]: They should probably still be able to see who has permissions (the list of addresses), just not the edit or add new button.

**Review by asgeir-eth — CHANGES_REQUESTED** [review]
I think this button must be removed when the account does not have `publication/permissions` permissions
image

File: packages/app/src/components/views/publication/components/PermissionSection.tsx (hunk 2 above)
Comment (juliopavila) [reply packages/app/src/components/views/publication/components/PermissionSection.tsx]: You're right!

**Review by asgeir-eth — APPROVED** [review]

**Pull request merged**